startup.sh

```diff
@@ -24,6 +24,9 @@ SECRETS_DIR=/opt/secrets
 # Docker then localize environment settings and insist on there
```

Comment:
GitHub won't let me comment on lines that it doesn't consider to fall within the limits of the changes (I hate that about GitHub).  However, line 20 of this file contains the actual name of the secrets directory, "`/opt/secrets`".

https://github.com/tl-its-umich-edu/kartograafr/pull/15/files#diff-d0486b0d7b181591f52bad014cd3a06fR21

Reply:
That's not a problem in a container.  Knowing that doesn't tell anyone where real, mapped, secret information really is outside of the container, and if someone is already inside the container all is lost already.  It's much the same if someone can see the environment running Docker.  If they can see that environment they already have access to things that aren't safe to see.

Reply:
My point was not about security or secrecy.  It's that the name of the directory is  "`/opt/secrets`", not "`./OPT/SECRETS`", as shown elsewhere.